src/main/java/uk/gov/hmcts/reform/sendletter/logging/AppInsights.java

```diff
@@ -1,13 +1,56 @@
 package uk.gov.hmcts.reform.sendletter.logging;
 
+import com.google.common.collect.ImmutableMap;
 import com.microsoft.applicationinsights.TelemetryClient;
+import com.microsoft.applicationinsights.telemetry.Duration;
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.reform.logging.appinsights.AbstractAppInsights;
 
+import java.util.Collections;
+import java.util.Map;
+
 @Component
 public class AppInsights extends AbstractAppInsights {
 
+    static final String SERVICE_BUS_DEPENDENCY = "ServiceBus";
+    static final String SERVICE_BUS_MESSAGE_ACKNOWLEDGED = "MessageAcknowledged";
+    static final String MESSAGE_RECEIVED = "MessageReceived";
+
     public AppInsights(TelemetryClient telemetry) {
         super(telemetry);
     }
+
+    public void trackMessageReceived(String serviceName, String template, String messageId) {
+        Map<String, String> properties = ImmutableMap.of(
+            "service",
+            serviceName,
+            "template",
+            template,
+            "messageId",
+            messageId
+        );
```

Comment:
that would be way easier to read imho:
```
       Map<String, String> properties = ImmutableMap.of(
           "service", serviceName,
           "template", template,
           "messageId", messageId
       );
```
I hope checkstyle will allow this

Reply:
didn't risk initially to not pester checkstyle :D